libs/slam/async_slam/async_slam_worker.cpp

```
@@ -0,0 +1,282 @@
+
+    const auto& frame_data = vo_kf->frame_data;
+    RERUN(visualizer::RerunVisualizer::getInstance().setupTimeline, frame_data.frame_id, frame_data.timestamp_ns);
+
+    // execute command from input_queue_
+    if (vo_kf->command) {
+      std::shared_ptr<ICommand>& cmd = vo_kf->command;
+      cmd->Execute(*this, end_frame_id, vo_pose_at_that_frame);
+      continue;
+    }
+
+    Images current_images;
```

Clang (14.0.6)

[warning] variable 'current_images' is not initialized

(cppcoreguidelines-init-variables)